src/ServiceFabric/ServiceFabric/Template/Windows/template.json

```diff
@@ -633,15 +638,15 @@
                     "clusterEndpoint": "[reference(parameters('clusterName')).clusterEndpoint]",
                     "nodeTypeRef": "[variables('vmNodeType0Name')]",
                     "dataPath": "D:\\SvcFab",
-                    "durabilityLevel": "[parameters('durabilityLevel')]",
+                    "durabilityLevel": "Silver",
```

Comment:
wondering if instead of removing this as a parameter we just dont allow bronze instead so that people can still make gold clusters

Reply:
updated